### ISSUE TITLE
Replace shebang?

### ISSUE THREAD

**Issue by kglotfelty** [cxcsds/ciao-contrib#128]
@DougBurke has mentioned this before and I'm just recording this here as something to be considered. I'm not necessarily advocating for it.  

Part of the issue related to  #127 is that we use the shebang

```
#!/usr/bin/env python
```
in the contrib scripts.

This is convenient but leaves us open to problems where users (or OTS) change the `PATH` such that CIAO's `python` is no longer first.  

*Q: Should we change to a hard-coded shebang that gets configured?*

That is in the contirb tar file we would ship `contrib/bin/foo.in` with a shebang macro

```
#!@PYTHON@
```

kind of thing.  Then CIAO's configure would need to replace that with the hard coded location to CIAO, eg

```
#!/soft/ciao/bin/python
```

> I think `bin` rather than `ots/bin` to deal with `.ciaorc` stuff?

This would require two changes

- Change all `contrib/bin` script to use the new token
- Change `configure` to do the macro expansion.

I actually thought this is what was happening with the DS scripts, eg acis_streak_map,  but it's not.  The macro is expanded when the script is built and installed, and for CIAO it sets it to the same generic `/usr/bin/env python` .  Oddly `arfcorr` is just hard-coded that way  (no `.in` file).

Thoughts?

**Comment by DougBurke** [comment]
I guess this is still an issue even with the `python setup.py install` work (it's less relevant for the conda builds I think since they do there own path munging).

**Comment by kglotfelty** [comment]
yes, it's still an issue because it bakes in the ots/bin/python path -- but
it needs the wrapper to set the env. variables.
